docs/core/diagnostics/distributed-tracing-collection-walkthroughs.md

```diff
@@ -0,0 +1,326 @@
+---
+title: Collect a distributed trace - .NET
+description: A tutorial to collect distributed traces in .NET applications
+ms.topic: tutorial
+ms.date: 03/14/2021
+---
+
+# Collect a distributed trace
+
+**This article applies to: ✔️** .NET Core 5.0 and later versions **or** any .NET application using the 
+[DiagnosticSource NuGet package](https://www.nuget.org/packages/System.Diagnostics.DiagnosticSource/5.0.1) version 5 or later
+
+.NET applications can be instrumented using the <xref:System.Diagnostics.Activity?displayProperty=nameWithType> API to produce
+distributed tracing telemetry. In this tutorial you will record this instrumented telemetry with different telemetry collection
+libraries so that it is available to diagnose application issues. See 
+[the instrumentation tutorial](distributed-tracing-instrumentation-walkthroughs.md) if you need to add new instrumentation.
+
+## Collect using OpenTelemetry
+
+### Prerequisites
+
+- [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet) or a later version
```

Comment:
Why 3.1 as the pre-req? This would work even in 2.1 right?

Reply:
Probably, I just didn't test it. I'm going to leave it as-is for now but if customers care we could re-test the workflow using an older SDK to ensure I didn't accidentally use a 3.1 only feature.